CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### Changed
+
+- Made error message for -11/-9 more actionable
```

Comment:
I think this could be made a little more approachable for those not already familiar with the error codes. Maybe something like "Made error message for resource exhaustion (code -11/-9) more actionable"